opennms-webapp-rest/pom.xml

```diff
@@ -399,6 +399,22 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.cxf</groupId>
+      <artifactId>cxf-rt-rs-service-description-openapi-v3</artifactId>
+      <version>3.2.4</version>
```

Comment:
version can be defined in parent pom.xml and can be referenced here.

Reply:
ah, good catch

Reply:
Good catch. Updated.